codespell_lib/_codespell.py

```diff
@@ -413,8 +413,15 @@ def parse_options(args):
         for key in config['codespell']:
             # Add option as arg.
             cfg_args.append("--%s" % key)
+            # strip comments from lines
+            filtered_lines = []
+            for line in config['codespell'][key].splitlines():
+                line = line.strip()
+                if line != "":
+                    filtered_lines.append(line)
+            # concatenate non-empty lines
+            val = ','.join(filtered_lines)
```

Comment:
This isn't a mere concatenation. Lines are separated by `,`.

This works with `skip`, will it work with `dictionary`?